rust/noosphere-ns/src/name_system.rs

```diff
@@ -17,6 +17,15 @@ use crate::NameSystemBuilder;
 #[cfg(doc)]
 use cid::Cid;
 
+pub static BOOTSTRAP_PEERS_ADDRESSES: [&str; 1] =
+    ["/ip4/134.122.20.28/tcp/6666/p2p/12D3KooWAKxaCWsSGauqhCZXyeYjDSQ6jna2SmVhLn4J7uQFdvot"];
```

Comment:
Do you think in time it may make sense to accept this list via environment or CLI configuration?

Reply:
These should be rather permanent and ubiquitous, and should be the default list of peers for most clients -- that being said, the name system (and individual DHT nodes) do not prescribe this list, and this bootstrap script passes this list along (filtering out its own address, some work needs to be done on wrangling the multiple forms of addresses here, and how they're referenced etc.).

Specifically on *customizing bootstrap peers via CLI/env*, that could be useful for the "run your own bootstrap nodes" flow, though no immediate need yet

Reply:
Acknowledging that in the typical, these peers should be long-lived and well-known, I think we may need the flexibility for this list to be customizable. Example use cases:

 - I have private infrastructure and don't want my nodes to bootstrap off of publicly available peers
 - I'm an infrastructure provider and my well-known peers are a dynamic detail of my cloud infrastructural configuration

No action required for this change though.